bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -24,6 +24,14 @@
 				<description>Password of tado login used for API access</description>
 				<context>password</context>
 			</parameter>
+
+			<parameter name="maxReconnectAttempts" type="integer" min="0" max="10" step="1" required="true">
+				<default>3</default>
+				<label>Max Reconnect Attempts</label>
+				<description>Maximum allowed number of attempts to reconnect things after going offline due to a communication error</description>
+				<advanced>true</advanced>
+			</parameter>
```

Comment:
i18n properties need to be rebuilt after adding these new strings.

Reply:
Ok. Will fix that.